src/utilities/getCategoryData.js

```diff
@@ -0,0 +1,32 @@
+const getCategoryData = (categoryDataList, categoryState) => {
+  const data = [];
+  //check the active category form categroyState
+  Object.entries(categoryState).forEach(([categoryName, categoryActive]) => {
+    // check the product category matches the active category
+    data.push(
+      ...categoryDataList.filter(({ productCategory }) => {
+        return categoryActive ? productCategory[categoryName] : false;
+      })
+    );
+  });
+
+  console.log(data);
```

Comment:
Once you are done with this file, can remove console.log from here

Reply:
Yeah thanks for the suggestion, will update the explore button😅